src/core/include/ngraph/util.hpp

```diff
@@ -258,7 +258,7 @@ NGRAPH_API_DEPRECATED std::vector<T> read_vector(std::shared_ptr<ov::Tensor> tv)
     if (ngraph::element::from<T>() != tv->get_element_type()) {
         OPENVINO_THROW("read_vector type must match Tensor type");
     }
-    size_t element_count = ngraph::shape_size(tv->get_shape());
+    size_t element_count = ov::shape_size(tv->get_shape());
```

Comment:
no actions required to this comment: looks like after we remove `ngraph` namespace, we will have a lot of explicit `ov::` prefixes for API, which is already in `ov` namespace. Maybe we can clean up it after all

Reply:
I've tried to add `ov::` where is required code where still `ngraph` is used as main namespace.

In general agree with clean-up.
I think many changes are related ONNX FE (as there is still `ngraph` used) and tests.
In `ngraph` code files it is temporary as files will be removed. For rest the clean-up can be done after migration.